### ISSUE TITLE
Not found on npm

### ISSUE THREAD

**Issue by binary64** [DimensionDev/eslint-plugin#1]
```
$ yarn add -D @dimensiondev/eslint-plugin
yarn add v1.22.17
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@dimensiondev%2feslint-plugin: Not found".
```

**Comment by septs** [comment]
https://github.com/DimensionDev/eslint-plugin/packages/1174685

**Comment by septs** [comment]
```
npm install --save-dev @dimensiondev/eslint-plugin --registry=https://npm.dimension.im
```
